presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -104,9 +105,31 @@ private DistributedQueryRunner(
             Optional<Path> baseDataDir,
             List<SystemAccessControl> systemAccessControls)
             throws Exception
+    {
+        this(defaultSession, nodeCount, extraProperties, coordinatorProperties, null, environment, additionalModule, baseDataDir, systemAccessControls);
+    }
+
+    private DistributedQueryRunner(
+            Session defaultSession,
+            int nodeCount,
+            Map<String, String> extraProperties,
+            Map<String, String> coordinatorProperties,
+            Map<String, String> backupCoordinatorProperties,
+            String environment,
+            Module additionalModule,
+            Optional<Path> baseDataDir,
+            List<SystemAccessControl> systemAccessControls)
+            throws Exception
     {
         requireNonNull(defaultSession, "defaultSession is null");
 
+        if (backupCoordinatorProperties != null) {
```

Comment:
If we are to determine if we should use backup coordinator based on existence of `backupCoordinatorProperties` then declare it as `Optional<Map<String, String>` and use `isPresent()` for check.